celi_framework/main.py

```diff
@@ -37,6 +39,34 @@
 
 logger = logging.getLogger(__name__)
 
+def get_default_config_path():
+    # Using importlib.resources to safely access the package resource
+    try:
+        config_path = resources.files('celi_framework.examples.wikipedia').joinpath('example_config.json')
+        # Temporary use during development:
+        # config_path = "/Users/jwag/PycharmProjects/celi/celi_framework/examples/wikipedia/example_config.json"
+        return str(config_path)
+    except Exception as e:
+        print(f"Failed to load default config: {e}")
+        return None
+
+def resolve_config_path(provided_path):
+    # Convert relative path to absolute if necessary and check existence
+    try:
+        path_obj = Path(provided_path)
+        if not path_obj.is_absolute():
+            # Assuming this script is run from the root of the project
```

Comment:
This is not an assumption you can make generally.
In a pip install the project isn't there at all.  The user would be configuring those files relative to the current directory in which they run (or use absolute paths).
This code only works if you are checking out the git repo and won't ever work for a general install.